app/domain/authentication/authn_jwt/jwt_configuration_interface.rb

```diff
@@ -0,0 +1,9 @@
+module Authentication
+  module AuthnJWT
+    class JWTConfigurationInterface
```

Comment:
can you please add a comment on this class?

Reply:
Added comment